zkevm-circuits/src/evm_circuit.rs

```diff
@@ -365,12 +365,14 @@ pub mod test {
         run_test_circuit(block)
     }
 
-    pub fn run_test_circuit<F: Field>(block: Block<F>) -> Result<(), Vec<VerifyFailure>> {
-        let fixed_table_tags = detect_fixed_table_tags(&block);
+    pub fn test_circuit_degree<F: Field>(block: &Block<F>) -> u32 {
```

Comment:
This function name `test_circuit_degree` (and so as `test_circuit_instance`, `test_cicuit_from_block`...) looks like test sth, but it turn out that it's like getting XXX for testing circuit, cause a bit confusing. How about rename to get_degree(_from_block), get_instance, gen_circuit...etc

Reply:
ok, I would do get_test_degree, get_test_instance and so on...